cmd/lncli/commands.go

```diff
@@ -1989,10 +1989,10 @@ func feeReport(ctx *cli.Context) error {
 	return nil
 }
 
-var updateFeesCommand = cli.Command{
-	Name:      "updatefees",
-	Usage:     "update the fee policy for all channels, or a single channel",
-	ArgsUsage: "base_fee_msat fee_rate [channel_point]",
+var updateChannelPolicyCommand = cli.Command{
```

Comment:
Hmm, wondering if I should just keep the name `updatefees` even though it is used for more than strictly setting fees, as it might be confusing that we change the name suddenly.

Reply:
Eh, this is more descriptive as it's no longer _just_ about fees as you note. I think the naming change would be an issue if we were publishing stable releases, but at this point, I don't think it'll critically affect anyone's deployed applications. 

Naming suggestion to shorten a bit: `updatechanpolicy`. Less to type on the command line 😛